README.md

```diff
@@ -1,11 +1,7 @@
 # API Enhancement Proposals
 
-**NOTE**: this repository currently has a significant amount of outdated
-content. Active development and discussion is occuring in
-[aepc](https://github.com/aep-dev/aepc).
-
-Once an API specification and design pattern guide solidifies, this repository
-will be updated.
+The [AEP Project](www.aep.dev) is an API design specification and ecosystem of
```

Comment:
```suggestion
The [AEP Project](https://aep.dev) is an API design specification and ecosystem of
```
www.aep.dev just redirects to aep.dev anyway, so I think it's worth skipping the redirect.